### PR TITLE
Iteration5.1

### PR DESCRIPTION
**Pull request by LeahWinters**
# Pull Request Template

## Description

When a saved outfit card is clicked:
     - it dresses the bear in that outfit
     - it updates the selected garment buttons to match
     - it fills the form with the name of the outfit

Completed as a team!

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
## How Has This Been Tested?

wWith console.logs and debuggers

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] My changes generate no new warnings

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -82,7 +84,7 @@ function saveOutfit() {
```
Comment (KmBarnett) [inline main.js:84]: lines 36 and 87 allow for cards to be clickable

File: main.js [hunk 2]
```diff
@@ -132,3 +134,18 @@ function removeSaveCard() {
   outfitList.splice(remove, 1);
   saveToLocalStorage();
 }
+
+function redressBearHelper() {
+  var outfitSelected = outfitList.find(outfit => outfit.id === event.target.id)
+  currentOutfit = new Outfit(outfitSelected);
+  var actions = [buttonHats[currentOutfit.garments[0].clickValue],
+  buttonClothes[currentOutfit.garments[1].clickValue],
```
Comment (KmBarnett) [inline main.js:142]: this is the code that redresses the bear by reclicking the buttons via javascript

File: outfit.js [hunk 3]
```diff
@@ -2,33 +2,35 @@ class Outfit {
   constructor(outfitFile) {
     this.title = outfitFile.title;
     this.id = outfitFile.id || this.generateIdNum();
-    this.garments = outfitFile.garments || [null, null, null];// NOTE: hats, clothes, accessories//
+    this.garments = outfitFile.garments || new Array(3).fill({garment: undefined, clickValue: undefined });
```
Comment (KmBarnett) [inline outfit.js]: this gives us a better array for functionality

File: outfit.js [hunk 4]
```diff
@@ -2,33 +2,35 @@ class Outfit {
...
     }
   }
 
   removeGarment(garment) {
     if (garment.classList.contains('hats')) {
-      this.garments.splice(0, 1);
+      this.garments.splice(0, 1, '');
     } else if (garment.classList.contains('clothes')) {
-      this.garments.splice(1, 1);
+      this.garments.splice(1, 1, '');
```
Comment (KmBarnett) [inline outfit.js]: i think the empty string needs to be replaced with {garment: undefined, clickValue: undefined}

File: outfit.js [hunk 5]
```diff
@@ -39,4 +41,6 @@ class Outfit {
   generateIdNum() {
     return Math.random().toString(36).slice(2, 8);
   }
+
+
```
Comment (KmBarnett) [inline outfit.js]: remove this extra space

**Review by KmBarnett — CHANGES_REQUESTED** [review]
some small changes please

File: outfit.js (hunk 4 above)
Comment (LeahWinters) [reply outfit.js]: Sounds great, we can for sure do that!

File: outfit.js (hunk 5 above)
Comment (LeahWinters) [reply outfit.js]: no problem! We can delete one of those spaces

**Comment by KmBarnett** [comment]
Looks great thanks for the changes Kyle and Jeremy navigated

**Pull request merged**